requirements/base.txt

```diff
@@ -231,6 +231,8 @@ parsedatetime==2.6
     # via apache-superset
 pgsanity==0.2.9
     # via apache-superset
+playwright==1.37.0
+    # via apache-superset
```

Comment:
I think this should probably go into the dev, integration or test deps

Reply:
Could you explain why? Isn't reports one of the basic flows? Or maybe I misunderstand the purpose of those files

Reply:
The fact that it's marked as an optional dependency in `setup.py` IMO means it should not be in `base.txt`. Was this entry added by hand, or via `pip-compile-multi`? I assume we'd need to change `requirements/base.in` as follows to autogenerate this:

```
-e file:.[playwright]
```